Patches/Mechanoid Bench 3 JGH/Apparel_MB3.xml

```diff
@@ -0,0 +1,297 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Patch>
+    <Operation Class="PatchOperationFindMod">
+        <mods>
+            <li>Mechanoid Bench 3</li>
+        </mods>
+        <match Class="PatchOperationSequence">
+            <operations>
+
+                <!-- ===== Add Bodypart Groups and Layers ===== -->
+                <li Class="PatchOperationAdd">
+                    <xpath>/Defs/ThingDef[@Name="ApparelMBHelmetBase"]/apparel/layers</xpath>
+                    <value>
+                        <li>OnHead</li>

```

Comment:
isn't OnHead the lower layer?

Reply:
it is according to other power armor helmet variant mod, the layer order is correct